vk_video_decoder/libs/VkCodecUtils/VulkanDeviceContext.cpp

```diff
@@ -167,17 +167,18 @@ bool VulkanDeviceContext::HasAllDeviceExtensions(VkPhysicalDevice physDevice, bo
     }
 
     // all listed device extensions are required
-    for (const auto &name : m_reqDeviceExtensions) {
-        if (ext_names.find(name) == ext_names.end()) {
-            if (printMissingExt) {
-                std::cerr << "HasAllDeviceExtensions() ERROR: requested device extension "
-                    << name << " is missing!" << std::endl << std::flush;
-            }
-            return false;
+    auto nameIterator = m_reqDeviceExtensions.begin();
+    while (nameIterator != m_reqDeviceExtensions.end()) {
+        if (ext_names.find(*nameIterator) == ext_names.end()) {
+            std::cout << " Extension " << *nameIterator << " is not supported" << std::endl;
+            nameIterator = m_reqDeviceExtensions.erase(nameIterator);
```

Comment:
We can't modify this list - this is the user requested list of extensions. If we can't satisfy the extensions list, it needs to be adjusted. What extension(s) was/were causing problem for AMD?

Reply:
VK_EXT_YCBCR_2PLANE_444_FORMATS_EXTENSION_NAME
VK_KHR_VIDEO_ENCODE_QUEUE_EXTENSION_NAME

These are not supported by AMD.

Reply:
> VK_EXT_YCBCR_2PLANE_444_FORMATS_EXTENSION_NAME VK_KHR_VIDEO_ENCODE_QUEUE_EXTENSION_NAME
> 
> These are not supported by AMD.

We can remove these extensions - NP.